src/institutions/container.jsx

```diff
@@ -17,8 +17,10 @@ export class InstitutionContainer extends Component {
     // create the expected objects from the array, institutions = [{lei: lie}, {lei: lei}]
     let institutions = []
     leis.forEach(lei => {
-      institutions.push({ lei: lei })
+      institutions.push({ lei: lei, year: 2018 })
+      institutions.push({ lei: lei, year: 2019 })
```

Comment:
I don't think this is appropriate here, we should use the values returned in the keycloak object to associate leis with institutions rather than create these associations here.